addOns/scanpolicies/src/main/javahelp/help/contents/policy-api.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+API Policy
+</TITLE>
+</HEAD>
+<BODY>
+<H1>API Policy</H1>
+
+A lighter policy focusing on issues likely to impact APIs and not UI.
```

Comment:
Do we really want to say "lighter"?
My worry is that people might then this its not so useful, whereas is actually all of the rules we think are suitable for APIs, right?

Reply:
Sure I'll tweak it